src/content/scripting/script.cc

```diff
@@ -228,7 +228,7 @@ Script::Script(std::shared_ptr<ContentManager> content,
     }
 
     duk_push_object(ctx); // config
-    for (int i = 0; i < int(CFG_MAX); i++) {
+    for (size_t i = 0; i < CFG_MAX; i++) {
         auto scs = ConfigDefinition::findConfigSetup(config_option_t(i), true);
```

Comment:
That one is based on an enum, so I think we should keep `int` instead.

Reply:
It's cast to config_option_t anyway.

Reply:
That's why I'm actually thinking about writing an iterator over `config_option_t`. Furthermore: it is not used as an array index.

Reply:
Removed.